src/backgrounds.rs

```diff
@@ -1,11 +1,74 @@
 use bevy::{prelude::*};
+use crate::player::{Player};
+use crate::wfc::wfc;
 
-pub(crate) const LEVEL_LENGTH: f32 = 5000.;
-pub(crate) const LEVEL_HEIGHT: f32 = 5000.;
-pub(crate) const TILE_SIZE: f32    = 16.  ;
+pub(crate) const TILE_SIZE: f32    = 64.  ;
+pub(crate) const MAP_WIDTH: usize  = 20   ;
+pub(crate) const MAP_HEIGHT: usize = 12   ;
+pub(crate) const WIN_H: f32        = 720. ;
+pub(crate) const WIN_W: f32        = 1280.;
+pub(crate) const LEVEL_WIDTH: f32 = MAP_WIDTH as f32 * TILE_SIZE;
+pub(crate) const LEVEL_HEIGHT: f32 = MAP_HEIGHT as f32 * TILE_SIZE;
 
-pub(crate) const GAME_BACKGROUND: &str = "backgrounds/test_scroll_background.png";
 
 #[derive(Component)]
 pub(crate) struct Background;
 
+pub(crate) fn init_background(mut commands: Commands,
+    asset_server: Res<AssetServer>,
+    mut texture_atlases: ResMut<Assets<TextureAtlas>>
+) {
+
+    let starting_chuck = wfc();
```

Comment:
Think you meant to name this `starting_chunk`

```suggestion
    let starting_chunk = wfc();
```